maquetado/scss/books.scss

```diff
@@ -43,7 +45,10 @@
 }
 
 .book-info-container {
+  display: grid;
   flex-grow: 1;
+  grid-template-rows: 0.2fr 0.2fr 0.2fr 0.2fr;
+  margin-top: 40px;
```

Comment:
Don't use margin to separate an inner element from its parent. Set `padding-top` here instead